packages/sit-onyx/src/styles/mixins/layers.scss

```diff
@@ -0,0 +1,11 @@
+@use "normalize.scss";
+
+@mixin component() {
+  @layer onyx.reset {
+    @include normalize.apply;
```

Comment:
Alternatively we could place the normalize styles itself in `onyx.reset` layer so we could remove this block and just define the below layers as `@layer onyx.reset onyx.component`

Reply:
thats not how layers work, layers only define the cascade not any rules